ioctl/cmd/account/accountactions.go

```diff
@@ -0,0 +1,202 @@
+var countLimit *uint64
+
+// Multi-language support
+var (
+	actionsCmdShorts = map[config.Language]string{
+		config.English: "Show the list of actions for an account",
+		config.Chinese: "显示账户的操作列表",
+	}
+	actionsCmdUses = map[config.Language]string{
+		config.English: "actions (ALIAS|ADDRESS)  [SKIP]",
+		config.Chinese: "actions (ALIAS|ADDRESS)  [SKIP]",
+	}
+	flagCountUsages = map[config.Language]string{
+		config.English: "choose a count limit",
+		config.Chinese: "选择一个计数限制",
+	}
+)
+
+// accountActionsCmd represents the account sign command
+var accountActionsCmd = &cobra.Command{
+	Use:   config.TranslateInLang(actionsCmdUses, config.UILanguage),
+	Short: config.TranslateInLang(actionsCmdShorts, config.UILanguage),
+	Args:  cobra.RangeArgs(1, 2),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		err := accountActions(args)
+		return output.PrintError(err)
+	},
+}
+
+func init() {
+	countLimit = accountActionsCmd.Flags().Uint64("count", 15, config.TranslateInLang(flagCountUsages, config.UILanguage))
+}
+
+func accountActions(args []string) error {
+	var skip uint64
+	var err error
+	if len(args) == 2 {
+		skip, err = strconv.ParseUint(args[1], 10, 64)
+		if err != nil {
+			return output.NewError(output.ConvertError, "failed to convert skip ", nil)
+		}
+	}
+
+	addr, err := util.Address(args[0])
+	if err != nil {
+		return output.NewError(output.AddressError, "failed to get address", err)
+	}
+
+	conn, err := util.ConnectToEndpoint(config.ReadConfig.SecureConnect && !config.Insecure)
+	if err != nil {
+		return output.NewError(output.NetworkError, "failed to connect to endpoint", err)
+	}
+	defer conn.Close()
+	cli := iotexapi.NewAPIServiceClient(conn)
+	ctx := context.Background()
+
+	jwtMD, err := util.JwtAuth()
+	if err == nil {
+		ctx = metautils.NiceMD(jwtMD).ToOutgoing(ctx)
+	}
+
+	requestGetAccount := iotexapi.GetAccountRequest{
+		Address: addr,
+	}
+	accountResponse, err := cli.GetAccount(ctx, &requestGetAccount)
+	numActions := accountResponse.AccountMeta.GetNumActions()
+	fmt.Println("Total:", numActions)
+	requestGetAction := iotexapi.GetActionsRequest{
+		Lookup: &iotexapi.GetActionsRequest_ByAddr{
+			ByAddr: &iotexapi.GetActionsByAddressRequest{
+				Address: addr,
+				Start:   numActions - *countLimit - skip,
+				Count:   *countLimit,
+			},
+		},
+	}
+	response, err := cli.GetActions(ctx, &requestGetAction)
+	if err != nil {
+		sta, ok := status.FromError(err)
+		if ok {
+			return output.NewError(output.APIError, sta.Message(), nil)
+		}
+		return output.NewError(output.NetworkError, "failed to invoke GetActions api", err)
+	}
+	if len(response.ActionInfo) == 0 {
+		return output.NewError(output.APIError, "no action info returned", nil)
+	}
+	showFields := []interface{}{
+		"Hash",
+		//"Time",
+		"Status",
+		"Sender",
+		"Type",
+		"To",
+		"Contract",
+		"Amount",
+	}
+	tbl := table.New(showFields...)
+
+	for k := range response.ActionInfo {
+		k = len(response.ActionInfo) - 1 - k
```

Comment:
this works yet a little confusing, suggest use 
```
for i := range
     k = len(response.ActionInfo) - 1 - i
```

Reply:
ok